src/ArticleLine.php

```diff
@@ -8,85 +8,76 @@
  */
 class ArticleLine
 {
+    private $freeText1;
```

Comment:
Didn't we have some traits for FreeText?

Reply:
Fixed in #142